Saule/JsonApiSerializer.cs

```diff
@@ -72,7 +72,9 @@ public PreprocessResult PreprocessContent(object @object, ApiResource resource,
                     type: resource,
                     baseUrl: requestUri,
                     urlBuilder: UrlPathBuilder,
-                    paginationContext: QueryContext?.Pagination);
+                    paginationContext: QueryContext?.Pagination,
+                    includingContext: QueryContext?.Including,
+                    includedDefault: QueryContext != null ? QueryContext.IncludedDefault : true);

```

Comment:
I understand why this is a parameter (query context may always be `null`), but I'd prefer if `null` was just interpreted as `true` in the code, with a comment. There's probably some edge case I can't think of where that doesn't work, but if it does then I'd prefer not adding an extra parameter.